### PR TITLE
Af4að

### PR DESCRIPTION
**Pull request by thorunna**

### CODE REVIEW

File: src/reynir/config/Verbs.conf [hunk 1]
```diff
@@ -1750,7 +1749,7 @@ gera             þgf nf  $score(-2)  # ???
 gera             þgf þf # 'að gera fólki skráveifu', 'að gera einhverjum óleik'
 gera			 /að verk_hk_ft_þgf
 gera			 /af verk_hk_ft_þgf	    $error(PP, /að verk_hk_ft_þgf)
-gera			 grín_hk_et_þf /að þgf
+gera			 grín_hk_et_þf /að þgf  $error(PP, /að þgf)
```
Comment (vthorsteinsson) [inline src/reynir/config/Verbs.conf:1752]: Something fishy here?

File: src/reynir/config/Verbs.conf [hunk 2]
```diff
@@ -3423,6 +3426,8 @@ leggja			 þf /af mark_hk_ft_þgf
 leggja			 þf /að mark_hk_ft_þgf	$error(PP, /af mark_hk_ft_þgf)
 leggja           þf /í eyði_hk_et_þf
 leggja           þf /í eiði_hk_et_þf        $error(PP-OBJ, eyði_hk_et_þf)
+leggja 			 þf /að völlur_kk_et_þgf
+leggja 			 þf /af völlur_kk_et_þgf 	$error(PP, /af völlur_kk_et_þgf) # 'leggja e-n að velli'
```
Comment (vthorsteinsson) [inline src/reynir/config/Verbs.conf:3430]: Correction should be /að, not /af

File: src/reynir/config/Verbs.conf [hunk 3]
```diff
@@ -3808,6 +3816,8 @@ margþakka        þgf /fyrir þf
 margþakka        þgf þf
 margþæfa         þf /í þf
 marka            /af þgf /fyrir þgf
+marka 			 upphaf_hk_et_þf /að þgf
+marka 			 upphaf_hk_et_þf /af þgf $error(PP, /af þgf)
```
Comment (vthorsteinsson) [inline src/reynir/config/Verbs.conf:3820]: Correction should be /að, not /af

File: src/reynir/config/Verbs.conf (hunk 1 above)
Comment (thorunna) [reply src/reynir/config/Verbs.conf:1752]: Oh right, I've fixed it!

File: src/reynir/config/Verbs.conf (hunk 2 above)
Comment (thorunna) [reply src/reynir/config/Verbs.conf:3430]: Done

File: src/reynir/config/Verbs.conf (hunk 3 above)
Comment (thorunna) [reply src/reynir/config/Verbs.conf:3820]: Done

**Comment by vthorsteinsson** [comment]
OK to merge @thorunna ?

**Comment by thorunna** [comment]
Yes!

**Pull request merged**